spring-boot/src/main/resources/application.yml

```diff
@@ -1,5 +1,10 @@
 server:
-  port: 8080
+  port: 8443
+  ssl:
+    key-store-type: <key store type>
```

Comment:
Please specify in README wht this attributes are and how to generate them

Reply:
I did update for the Readme file.